server/RenderContext.cs

```diff
@@ -626,9 +626,37 @@ public void Render(XGraphics graphics)
                     }
                 }
                 timers.Add(new Timer("droyne"));
-#endregion
+                #endregion
+
+                #region droyne
+                //------------------------------------------------------------
+                // Non-Hiver Worlds
+                //------------------------------------------------------------
+                if (styles.nonHiverWorlds.visible)
+                {
+                    solidBrush.Color = styles.nonHiverWorlds.textColor;
+                    foreach (World world in selector.Worlds)
+                    {
+                        if (world.HasCode("Nh"))
```

Comment:
I guess that depends whether Nh means the same as Native Intelligent Life, or a significant non-Hiver population? How does T5SS cope with NIL from T5 world generation?